toolchain/gcc.py

```diff
@@ -52,13 +52,12 @@ def setUp(self):
                              'gmp-devel', 'glibc-devel', 'mpfr-devel',
                              'makeinfo', 'texinfo', 'mpc-devel'])
         else:
-            packages.extend(['glibc-static', 'autogen', 'guile',
-                             'guile-devel', 'libgo', 'libgo-devel',
-                             'libgo-static', 'elfutils-devel',
+            packages.extend(['glibc-static', 'guile-devel', 'elfutils-devel',
```

Comment:
This would still fail complaining about guile-devel package.  Move this package to if(rhel8)